src/pages/api/index.tsx

```diff
@@ -0,0 +1,266 @@
+import {
+    ActionLink,
+    BlogArticle,
+    Button,
+    CodeBlock,
+    theme,
+    UiDependencyProvider,
+} from '@apify-packages/ui-library';
+import Link from '@docusaurus/Link';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Layout from '@theme/Layout';
+import ThemedImage from '@theme/ThemedImage';
+import React from 'react';
+import GitHubButton from 'react-github-btn';
+import styled from 'styled-components';
+
+import styles from './styles.module.css';
+import Hero from '../../components/Hero/Hero';
+import Section from '../../components/Section/Section';
+import Tabs from '../../components/Tabs';
+
+const HeroWrapper = styled(Hero)`
+    background-image: url("/img/hero_background.svg");
+`;
+
+const SectionWrapper = styled(Section)`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    gap: ${theme.space.space24};
+
+    @media (min-width: ${theme.layout.desktop}) {
+        flex-direction: row;
+    }
+
+    .Description {
+        display: flex;
+        flex-direction: column;
+        gap: ${theme.space.space24};
+    }
+
+    .DescriptionLinks {
+        display: flex;
+        gap: ${theme.space.space16};
+
+        a {
+            font-weight: bold;
+            text-wrap: nowrap;
+            max-height: 36px;
+        }
+    }
+
+    .MainSectionContent {
+        display: flex;
+        flex-direction: column;
+        gap: ${theme.space.space16};
+
+        a {
+            text-wrap: nowrap;
+            font-weight: bold;
+            font-size: 14px;
+            max-height: 36px;
+        }
+    }
+`;
+
+const RelatedArticlesWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 2.4rem;
+
+    @media (min-width: ${theme.layout.desktop}) {
+        flex-direction: row;
+    }
+
+    a {
+        width: 100%;
+    }
+`;
+
+const ClientCodeWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: ${theme.space.space24};
+
+    @media (min-width: ${theme.layout.desktop}) {
+        max-width: 50%;
+    }
+`;
+
+const TabTitleWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    gap: ${theme.space.space8};
+`;
+
+const BlogImageWrapper = styled.img`
+    border-radius: 12px;
+    height: 100%;
+`;
+
+export default function Api() {
+    return (
+        <UiDependencyProvider dependencies={{
+            InternalLink: (props) => <Link {...props} />,
+            windowLocationHost: useBaseUrl(''),
+            isHrefTrusted: () => true,
+        }}>
+            <Layout>
+                <HeroWrapper
+                    heading="Apify API"
+                    isCentered
+                    description={<>Apify API provides useful features like automatic retries and convenience functions.<br/>
+                It connects you with any Actors and their tasks either in the Store or your own.</>}
```

Comment:
The copy doesn't make sense here, I know it's in the design but we need to change it to something connected to API in general.

I'd probably do something short and simple like:

Apify API provides programmatic access to the [Apify platform](https://docs.apify.com/).

What do you think?